packages/nimble-components/src/utilities/models/converter.ts

```diff
@@ -0,0 +1,17 @@
+import { booleanConverter, ValueConverter } from '@microsoft/fast-element';
+
+export const optionalBooleanConverter: ValueConverter = {
+    toView(value: unknown): string {
+        return typeof value === 'boolean' ? value.toString() : '';
```

Comment:
empty string isn't a valid option right? won't this result in `<my-el custom-hour12>` which is none of absent, "true", or "false"

```suggestion
        return typeof value === 'boolean' ? value.toString() : null;
```
Is there a testing gap?

Reply:
I've updated our test to fail with the current code and pass with the change.